### ISSUE TITLE
[Bug]: MetaPixel Errors Automatically Logged, Preventing Custom Error Handling

### ISSUE THREAD

**Issue by Sabourin01** [combindma/laravel-facebook-pixel#8]
### What happened?

When a MetaPixel request is sent and fails, the error is automatically logged instead of being thrown. This behavior prevents us from catching the error and handling it programmatically. Additionally, we would like to prevent these errors from always being logged in our error tracker.

### How to reproduce the bug

Any throwned exception is automatically logged when using the send function of the MetaPixel class.
![CleanShot 2024-11-25 at 09 14 12](https://github.com/user-attachments/assets/a9ecb2c6-9e57-4850-9535-1f73dea00d72)

### Package Version

5.0

### PHP Version

8.2

### Laravel Version

10.17

### Which operating systems does with happen with?

macOS

### Notes

It would be nice if there was a config that would let package either throw or log the error

**Comment by stefanocurnis** [comment]
Yes, this is breaking my Sentry account too. There's no way to bypass the logging.

**Comment by n4ss1m** [comment]
I think theses lines 

```PHP
$api->setLogger(new CurlLogger()); // curl logger to debug the request

// and

Log::error($e);
```

need to be wrapped with 

```PHP
if(config('app.debug')) {
   // ..
}
```

**Comment by omarherri** [comment]
Thank you for your suggestions! In release v5.0.3, I added the ability to enable or disable logging, which is now disabled by default. Please note, there should be no errors if everything is configured correctly. If errors were logged in the past, it likely indicates something is missing or misconfigured.
